### PR TITLE
kane ai api testing

### PR DESCRIPTION
**Pull request by Aman1905**

### CODE REVIEW

File: docs/kane-ai-api-testing.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image8.jpg').default} alt="kenai-jira integration" className="doc_img"/>
+  
+
+## 5. Adding Multiple APIs in One Go
+
+KaneAI allows batch processing of multiple APIs to streamline testing. This feature is helpful for scenarios requiring the execution of several API calls in succession.
+
+- **Step**: Add multiple APIs by clicking the plus icon and selecting each API, or paste multiple curl commands to add them automatically to the test steps.
+
+  <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image9.jpg').default} alt="kenai-jira integration" className="doc_img"/>
```
Comment (Private1647) [inline docs/kane-ai-api-testing.md:89]: This will be image 10.

File: docs/kane-ai-api-testing.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+## 6. Handling Different HTTP Methods
+
+KaneAI supports various HTTP methods like POST, PUT, GET, and DELETE, allowing you to test diverse API interactions.
+
+- **Step**: Add APIs using different HTTP methods, such as a PUT or DELETE command. Validate each API as before, and if successful, they will be automatically included in the test steps.
+
+  
+    <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image6.jpg').default} alt="kenai-jira integration" className="doc_img"/>
+  
+    <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image10.jpg').default} alt="kenai-jira integration" className="doc_img"/>
```
Comment (Private1647) [inline docs/kane-ai-api-testing.md:101]: This will be image 9.

File: docs/kane-ai-api-testing.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+## 7. Executing and Reviewing Test Steps
+
+Once all APIs are added, KaneAI enables simultaneous execution, with details available on methods used, response statuses, and execution times.
+
+- **Step**: Click to execute all added APIs in one go and review the response details for insights into API performance and data returned.
+
+    <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image10.jpg').default} alt="kenai-jira integration" className="doc_img"/>
```
Comment (Private1647) [inline docs/kane-ai-api-testing.md:109]: This needs to be have image 13

**Review by Private1647 — CHANGES_REQUESTED** [review]

File: docs/kane-ai-api-testing.md (hunk 3 above)
Comment (Aman1905) [reply docs/kane-ai-api-testing.md:109]: @Private1647 but you have only sent me 10 images,

**Pull request merged**